### DISCUSSION TITLE
sap_hana_preconfigure: No longer validate the RHEL minor version against a list of supported versions for SAP HANA

### DISCUSSION THREAD

**Discussion by berndfinger** [sap-linuxlab/community.sap_install#614] [Ideas]
It seems that this RHEL minor version check does not have much benefit to users.

Experience has shown that the minimum RHEL version check in role `sap_hana_preconfigure` caused some questions during the time when the roles are not yet in sync with the SAP notes. By removing the check, we could safely add the SAP HANA requirements during the prevalidation phase (if necessary update them after the related SAP note has been published) and users would no longer have to disable the minimum RHEL version check during that time.
Even the new code in the task

[[community.sap_install/roles/sap_hana_preconfigure/tasks/RedHat/installation.yml](https://github.com/sap-linuxlab/community.sap_install/blob/cc9e20ebf0e7b725c888d1b6924af5bc34a3ec56/roles/sap_hana_preconfigure/tasks/RedHat/installation.yml#L7)](https://github.com/sap-linuxlab/community.sap_install/commit/cc9e20ebf0e7b725c888d1b6924af5bc34a3ec56)

may not be useful and may lead to more questions.

Any comments on this?

Note: This topic first appeared in https://github.com/sap-linuxlab/community.sap_install/issues/581.

**Comment by sean-freeman** [comment]
I believe this check is necessary, ensuring end-users are informed / warned during an execution of the Ansible Role. It is better to inform an end-user to verify, than cause the end-user to potentially begin building an SAP System that may have an OS version without support / an OS version that may never be intended for support.

However, I acknowledge this may cause an end-user to panic if the Ansible Role release (e.g.  0.0.1) was updated (e.g. 0.0.2), and they did not update to that latest Ansible Role release but have executed the code on an OS release that is supported and codified as so in the later Ansible (e.g. 0.0.2).

This is a subjective choice. Either way, someone will be unhappy with the choice.
